### ISSUE TITLE
Replace windowrule with windowrulev2.

### ISSUE THREAD

**Issue by poffomoe** [hyprwm/Hyprland#3986]
### Description

I really enjoy using Hyprland and love how configurable it is. However, I'm confused by the fact that there are two versions of windowrule in config: windowrule (V1) and windowrulev2 (V2). V2 is simply better. It is more flexible, while remaining intuitive. So wouldn't it be better to replace V1 with V2? If not, why?

**Comment by vaxerski** [comment]
cuz breaks configs

**Comment by trainerlord** [comment]
I agree cause the syntax is almost exactly the same between the 2 with v2 allowing for more flexibility. All my rules are using v2

**Comment by poffomoe** [comment]
It's finally been replaced in 0.48! Thank you. 💖
